### ISSUE TITLE
Needs proper name

### ISSUE THREAD

**Issue by karimsa** [jaydenwindle/decode2017#47]
We need to come up with some sort of creative name for the project; "decode2017-shopify-thing" just isn't working out.

**Comment by jaydenwindle** [comment]
Agreed. I've been brainstorming a bit but haven't come up with any winners. Is it worth it to change the repo name as well? Everyone working on it would have to update their remotes, but that's not a huge deal

**Comment by jaydenwindle** [comment]
Currently leaning towards something like "Simple Giveaways" or "Simple Sweepstakes"

**Comment by karimsa** [comment]
I like going for just "Sweepstakes".

**Comment by JKirkYuan** [comment]
Shopaways

**Comment by vikrambombhi** [comment]
I was thinking something following raffles or sweepstakes
